deploy/crds/compute-node.openstack.org_v1alpha1_computenodeopenstack_cr.yaml

```diff
@@ -11,7 +11,16 @@ spec:
   k8sServiceIP: 172.30.0.1
   apiIntIP: 192.168.111.5
   workers: 0
-  corePinning: "4-7"
+  compute:
+    novaComputeCPUDedicatedSet: "4-7"
+    novaComputeCPUSharedSet: "0-3"
```

Comment:
we probably should pass through commonConfigMap and ospSecrets as those should get created by a controller operator in the end. neutron-operator will need them as well when ovn got added there

Reply:
those could be different for the different worker-osp computes right? I think it cannot be in the commons one. Also, idea is that computenode operator will also trigger the neutron CRs, right?

Reply:
I not meant to add novaComputeCPUDedicatedSet and novaComputeCPUSharedSet to the commonConfigMap. I meant instead of hard coding them in [1] to add commonConfigMap and ospSecrets to the CRD here and specify them in an upper level.

Yes it should also trigger the neutron CRs

[1] https://github.com/openstack-k8s-operators/compute-node-operator/commit/6f2587008650876b067ad53b6f804e6f359f4d35#diff-9ff7bfcbfc4889aa3d5f109ef58c9401